src/compat/libc/math/stub.c

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file stub.c
```

Comment:
This implementation should not be used in any circumstances, right? Why should it exist?

Reply:
@AntonKozlov It's for future porting on E2K :)